### PR TITLE
feat: a bit ugly, but closer

### PR DESCRIPTION
**Pull request by CVPhelan**
dont touch this yet, im still screwing with it

### CODE REVIEW

**Review by crookedneighbor — COMMENTED** [review]
This is looking good!

Merged in `main` and made a few adjustments (mostly changed the color from primary to white and converted the css you wrote to Tailwind classes)

File: nuxt.config.js [hunk 1]
```diff
@@ -83,7 +83,11 @@ export default {
   axios: {},
 
   // Build Configuration (https://go.nuxtjs.dev/config-build)
-  build: {},
+  build: {
+    options: {
+      fix: true,
```
Comment (crookedneighbor) [inline nuxt.config.js:88]: I couldn't find documentation on this. Can you provide a link?

Comment (CVPhelan) [reply nuxt.config.js:88]: https://stackoverflow.com/a/54210573

Comment (crookedneighbor) [reply nuxt.config.js:88]: I don’t think the way you configured it here actually does anything. 

You included the `options: { fix: true }` bit, but not as part of a modification for the eslint loader config. Let’s remove this for now, and if you need to include this for your set up, we can do it as a separate PR.

**Pull request merged**